### PR TITLE
Fix build failure

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/3496

## Description 

**Build Failure**

> Note: [NATS streaming server](https://github.com/nats-io/nats-streaming-server#warning--deprecation-notice-warning), java NATS streaming client, [NATS streaming server docker](https://hub.docker.com/_/nats-streaming) are all deprecated and will be dead by June next year. 

The TLS tests were failing. Tried the following. 
- Updating crypto dependency and certificates - didn't work 
- Disabled tests - reduced cod cov to 75%
- Changed docker image used by TLS server from latest to something that worked before (0.24.6) - this worked 

> Note: Should add a deprecation notice to ballerinax/stan as well. [Issue](https://github.com/ballerina-platform/ballerina-standard-library/issues/3497). 

**Trivy Failure**
Java dependency is the last version they have released and they haven't released anything after 2.2.3. https://github.com/nats-io/stan.java/releases. Therefore can't upgrade this. Not sure what to do about this. Vulnerability warnings are shown here too https://mvnrepository.com/artifact/io.nats/java-nats-streaming/2.2.3. Trivy might fail indefinitely.

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec

### CODE REVIEW

**Comment by dilanSachi** [comment]
Trivy failure is due to `protobuf` version right? Not `java-nats-streaming-2.2.3`

**Comment by aashikam** [comment]
> Trivy failure is due to `protobuf` version right? Not `java-nats-streaming-2.2.3`
used by it @dilanSachi 
Screenshot 2022-10-12 at 20 45 09

**Comment by dilanSachi** [comment]
That version is 3.9.1 right? But the trivy scan fails with 3.20.1. Cant we update it into 3.20.3 in Ballerina.toml

**Comment by aashikam** [comment]
> That version is 3.9.1 right? But the trivy scan fails with 3.20.1. Cant we update it into 3.20.3 in Ballerina.toml

Updated in 3d7bdfa02a11417a33373359131f513f26a03550

File: build.gradle [hunk 1]
```diff
@@ -28,7 +28,7 @@ ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.slf4jVersion = "1.7.30"
 ext.natsVersion = "2.8.0"
 ext.natsStreamingVersion = "2.2.3"
-ext.protobufVersion = "3.20.1"
+ext.protobufVersion = "3.20.3"
```
Comment (ThisaruGuruge) [inline build.gradle:31]: Better to move these versions to the gradle.properties file. Isn't it hard to maintain multiple files for maintaining versions?

File: build.gradle [hunk 2]
```diff
@@ -28,7 +28,7 @@ ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
...
 ext.stdlibCryptoVersion = project.stdlibCryptoVersion
 ext.stdlibLogVersion = project.stdlibLogVersion
 ext.stdlibTimeVersion = project.stdlibTimeVersion
```
Comment (ThisaruGuruge) [inline build.gradle:34]: We don't need these `ext` definitions.

File: build.gradle (hunk 1 above)
Comment (aashikam) [reply build.gradle:31]: This module is actually de-prioritised in every way. I've spent almost a day of JetStream time on this already. We will anyway have to deprecate this after JetStream. Shall we do this some other time 😅. Will do this if I have some time while doing JetStream. I'll create an issue. Is that okay.

Comment (ThisaruGuruge) [reply build.gradle:31]: +1

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**